iot-hub/Samples/module/ModuleSample/ModuleSample.cs

```diff
@@ -0,0 +1,265 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Azure.Devices.Client.Exceptions;
+using Microsoft.Azure.Devices.Shared;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.Devices.Client.Samples
+{
+    public class ModuleSample
+    {
+        private static readonly Random s_randomGenerator = new Random();
+        private const int TemperatureThreshold = 30;
+
+        private static readonly TimeSpan s_sleepDuration = TimeSpan.FromSeconds(5);
+        private static readonly TimeSpan s_operationTimeout = TimeSpan.FromHours(1);
+
+        private readonly object _initLock = new object();
+        private readonly List<string> _moduleConnectionStrings;
+        private readonly TransportType _transportType;
+
+        private readonly ILogger _logger;
+        private static ModuleClient s_moduleClient;
+
+        private static ConnectionStatus s_connectionStatus;
+        private static bool s_wasEverConnected;
+
+        public ModuleSample(List<string> moduleConnectionStrings, TransportType transportType, ILogger logger)
+        {
+            _logger = logger;
+
+            // This class takes a list of potentially valid connection strings (most likely the currently known good primary and secondary keys)
+            // and will attempt to connect with the first. If it receives feedback that a connection string is invalid, it will discard it, and
+            // if any more are remaining, will try the next one.
```

Comment:
I believe 'any more' should be 'anymore'

Reply:
any more = some more, quantitatively = would you like any more apples; 
anymore = relating to time = I don't like apples anymore.

so in this context, it should be "any more" => any more connection strings are remaining...